response_operations_ui/controllers/uaa_controller.py

```diff
@@ -94,6 +94,27 @@ def get_user_by_email(email, access_token=None):
     return response.json()
 
 
+def get_user_by_id(user_id, access_token=None):
+    if access_token is None:
+        access_token = login_admin()
```

Comment:
I am not sure I get this can you please let me know what does this do?

Reply:
I've tweaked this, but all that login_admin function does is get the token that allows response ops (as the client) do stuff in uaa.  It's not perfect, but it's lifted from the other stuff that seems to work.